files/his-dev/docker-compose.yml

```diff
@@ -299,22 +297,23 @@ services:
     restart: unless-stopped
     volumes:
       - eip-home-senaite:/eip-home
+
+  # Nginx Proxy
   proxy:
+    environment:
+      - GITPOD_ENV=${GITPOD_ENV}
     restart: unless-stopped
     image: openmrs/distro-his-proxy:latest
     healthcheck:
-      test:
-        - CMD
-        - curl
-        - "-f"
-        - "http://localhost/"
+      test: ["CMD", "curl", "-f", "http://localhost/"]
     networks:
       ozone:
     ports:
       - "8080:80"
       - "8069:8069"
       - "8081:8081"
-version: "3.7"
+    volumes:
+      - "${PROXY_TLS_CERTS_PATH:-proxy-tls-certs}:/etc/tls"
```

Comment:
Does this work if we don't have TLS certificates? Currently TLS is terminated at the host before it reaches the Docker stack.

Reply:
> Does this work if we don't have TLS certificates?
Yes, it works without certs